### DISCUSSION TITLE
An option to have alternative fighters on the same character slot.

### DISCUSSION THREAD

**Discussion by juney2008** [ikemen-engine/Ikemen-GO#1032] [Feature Request]
### Describe the solution you'd like

So I had this idea where you can add an character to the same character slot that is on there and be able to switch to the other fight like changing to an echo fighter in smash bros. I think it can make things easier to be able to add multiple versions of the same character without fulling out more space of the character select slots.

**Comment by K4thos** [comment]
this has been already implemented long time ago, refer to select.def distributed with the engine, slot parameter

**Reply by juney2008** [reply]
Oh, I never knew about that! Thanks for letting me know!
